vendor/extensions/taggings/app/controllers/refinery/taggings/taggings_controller.rb

```diff
@@ -13,11 +13,13 @@ def index
           .map {|t| TaggingSerializer.new(t).serializable_hash }
 
         respond_to do |f|
-          f.html { present(@page) }
+          f.html {
+            present(@page)
+          }
           f.json { render json: {
             taggings: filtered_taggings_json,
             topic_area_narrative: topic_area_narrative,
-            next_three_events: Refinery::Events::Event.next_three_events
+            next_three_events: Refinery::Events::Event.next_three_events,
```

Comment:
I would double check whether Ruby allows comma dangle, I know its good in JS but haven't tried for Ruby hash structures yet.

Reply:
Oh nevermind, that is in fact JSON.